apps/catalogue/app/composables/useHeaderData.ts

```diff
@@ -41,6 +42,12 @@ export async function useHeaderData() {
               Networks_agg: Resources_agg(filter:$networkFilter) {
                   count
               }
+              _settings (keys: [
+                "CATALOGUE_LOGO_SRC"
```

Comment:
we might want to use the existing molgenis logo setting? Or do we prefer to separate?

Reply:
I propose we move all these settings to the catalogue table, so we create also a special 'all' or 'default'  Then we can set logo, theme, description, etc, on per catalogue. This also allows different catalogues to be hosted on our server without need to configure the server but allowing data manager to set this themselves.